components/user-mgt/org.wso2.carbon.identity.user.pre.update.password.action/src/main/java/org/wso2/carbon/identity/user/pre/update/password/action/internal/execution/PreUpdatePasswordRequestBuilder.java

```diff
@@ -167,7 +167,7 @@ private PreUpdatePasswordEvent.Action getAction() throws ActionExecutionRequestB
             case PROFILE_UPDATE:
```

Comment:
With the flow stack we should be able to get rid off from this PROFILE_UPDATE type.

Reply:
Anyway let's fix that later